fastlane.gemspec

```diff
@@ -86,7 +86,7 @@ Gem::Specification.new do |spec|
   spec.add_dependency('json', '< 3.0.0') # Because sometimes it's just not installed
   spec.add_dependency('mini_magick', '>= 4.9.4', '< 5.0.0') # To open, edit and export PSD files
   spec.add_dependency('multi_xml', '~> 0.5')
-  spec.add_dependency('rubyzip', '>= 1.3.0', '< 2.0.0') # fix swift/ipa in gym
+  spec.add_dependency('rubyzip', '~> 2.2.0') # fix swift/ipa in gym
```

Comment:
We can't accept locking `rubyzip` to `2.2.0` and up due to the `rubyzip` requirement of Ruby 2.4. We could possibly accept `< 2.2.0` it it doesn't break anything 🤔

Reply:
It looks like my comments are not being read nor understood.

What part from my previous comments regarding the security fix on `rubyzip 2.0.0` and `ruby 2.3.8` is no longer maintained?

Reply:
It does not matter what is not maintained. Users of this library are using it on these versions, so we support these versions. We can not upgrade to a library version that does not support the versions of Ruby some users are using.

Reply:
@janpio is correct. We understand that there older versions have security issues and are not necessarily maintained. Updating to this would be a breaking change for a lot of users that are using older versions of Ruby that **fastlane still needs to support** and **that are still available for download**.

I would like it if could bump up the minimum required version of _fastlane_ but we cannot do it at this time **unless...**  `rubyzip 2.2.0` isn't a breaking change for us. 

If we could `spec.add_dependency('rubyzip', '>= 1.3.0', '< 2.2.0')` without _fastlane_ breaking that would allow us to **support older versions of Ruby** while allowing users to update to **that latest and securest dependencies**. But I don't know if the APIs for `rubyzip` between `1.3.0` and `2.2.0` remained unchanged that wouldn't break any way that we use it.